llvm/utils/git/github-automation.py

```diff
@@ -616,22 +616,20 @@ def create_pull_request(
     def execute_command(self) -> bool:
         """
         This function reads lines from STDIN and executes the first command
-        that it finds.  The 2 supported commands are:
-        /cherry-pick commit0 <commit1> <commit2> <...>
-        /branch <owner>/<repo>/<branch>
+        that it finds.  The supported command is:
+        /cherry-pick< ><:> commit0 <commit1> <commit2> <...>
         """
         for line in sys.stdin:
             line.rstrip()
-            m = re.search(r"/([a-z-]+)\s(.+)", line)
+            m = re.search(r"/([a-z-]+)\s*:? *(.*)", line)
```

Comment:
Shouldn't this be something like this now (and drop command below)?
```suggestion
            m = re.search(r"/cherry-pick\s*:? *(.*)", line)
```
As written, it looks to me like you will accept `/foobar commit` as well.

Reply:
Yes, I removed the if statement but did not change the regex appropriately. Also fixed another issue in the args command.